halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/config/v1/HalconfigParser.java

```diff
@@ -74,10 +72,15 @@ String halyardVersion(@Value("${Implementation-Version:unknown}") String version
   }
 
   @Bean
-  String spinconfigBucket(@Value("${spinnaker.config.bucket:halconfig}") String spinconfigBucket) {
+  String spinconfigBucket(@Value("${spinnaker.input.config.bucket:halconfig}") String spinconfigBucket) {
```

Comment:
Why not just use `@Value` on the fields? Are these consumed by something else? If they are, I'd suggest refactoring this into a `_Config` file and just autowiring that here (and the other places it's necessary) to keep organized.